foia_hub/settings/dev.py

```diff
@@ -15,6 +15,8 @@
 except ImportError:
     pass
 
+DOC_URL = ''
```

Comment:
@khandelwal ok, so I added some documentation, but this is the whole story. Maybe you have a better solution:
Basically, django storages was not designed to work with two buckets; however, this can be hacked by [forcing](https://github.com/projectkrang/foia-hub/blob/preparing_for_cf/docusearch/models.py#L52) the models to use another storage bucket. Unfortunately, when the document URL is rendered in the template, django treats it as a static file and defaults to the static bucket location. Adding in a doc URL overrides the static files's bucket default.